test/lib/oauth.js

```diff
@@ -382,7 +397,8 @@ var setupAppConfig = function(config, callback) {
 
     var dummy = {
         close: function() {
-            child.kill();
+            child.kill("SIGKILL"); // SIGKILL really needed??
+            child.disconnect();
```

Comment:
https://nodejs.org/docs/v0.10.25/api/child_process.html#child_process_child_kill_signal
Send a signal to the child process. If no argument is given, the process will be sent 'SIGTERM'
But that kept some code lingering (not finishing the test), the SIGKILL made sure the child was done.